huaweicloud/provider_test.go

```diff
@@ -130,6 +132,23 @@ func testAccPreCheckELB(t *testing.T) {
 	}
 }
 
+func testAccPreCheckCCE(t *testing.T) {
+	testAccPreCheckRequiredEnvVars(t)
+
+	if OS_VPC_ID == "" {
+		t.Skip("OS_VPC_ID must be set for CCE acceptance tests")
+	}
+	if OS_SUBNET_ID == "" {
+		t.Skip("OS_SUBNET_ID must be set for CCE acceptance tests")
+	}
+}
+
+func testAccPreCheckCCENode(t *testing.T) {
+	testAccPreCheckRequiredEnvVars(t)
+	if OS_SSH_KEY == "" {
+		t.Fatal("OS_SSH_KEY must be set for acceptance tests")
```

Comment:
better to also use t.Skip here as above  for such new added ENV variables in case it breaks all tests.